metron-interface/metron-config/src/app/sensors/reducers/sensors.reducers.ts

```diff
@@ -0,0 +1,638 @@
+/**
```

Comment:
@ruffle1986 I feel like we should break down this large reducer file into multiple files, as well as extract out the interfaces. I don't think it is necessarily an item that needs to be fixed for this PR to be accepted, but I do think it would make it easier to digest this code. What do you think?

Reply:
Good point. We can refactor it later once the feature is validated and accepted by users.